python/sphinx_docs/docs/conf.py

```diff
@@ -0,0 +1,44 @@
+# Configuration file for the Sphinx documentation builder.
+
+project = "opm.simulators"
+copyright = "2024 Equinor ASA"
+author = "Håkon Hægland"
+release = "0.1"
```

Comment:
shouldn't this be generated/derived from versioning information? seems a bit counter-productive to have a fixed version number completely separate from everything else.

Reply:
Yes maybe. Are you referring to the opm-simulators version like "2024.04" ?

Reply:
yes, which has a single source; dune.module in top of repository.

Reply:
Ok great. I have updated the script to read the version from dune.module